src/ucm/mmap/install.c

```diff
@@ -42,7 +43,8 @@
         _call;                                                                \
         ucm_trace("after %s: got 0x%x/0x%x", UCS_PP_MAKE_STRING(_call),       \
                   (_data)->fired_events, exp_events);                         \
-        (_data)->out_events &= ~exp_events | (_data)->fired_events;           \
+        (_data)->out_events &= ~(exp_events ^                                 \
+                                 ((_data)->fired_events & exp_events));       \
```

Comment:
i think this expression is equivalent to the previous one

Reply:
yes, but more "readable"...
will revert & add comment :)